Automattic/woocommerce-payments
# Add setting to disable customer multi currency feature

Fixes #2906

#### Changes proposed in this Pull Request


Add a setting to *WooCommerce > Payments* page, Advanced settings section to enable toggling the status of Customer Multi Currency feature 


This setting enables the merchant to disable customer multi currency feature bundled with WooCommerce Payments plugin, to prevent conflicts with other multi currency plugins the merchant is already using. This occured in one of the tickets we received (4283071-zd-woothemes) and discussed in thread p1631274038411900-slack-C01B8KNUYSW which @jessepearson created a gist (https://gist.github.com/jessepearson/0a4a2da18bdeef20b80b50ff0e727c61) for the user to temporarily fix the issue. This PR is removing the need for that code to be added to user's `functions.php` file.




The setting looks like:

![image](https://user-images.githubusercontent.com/3295/133255378-12bdefa7-3083-440f-8675-8623f47c696b.png)

#### Testing instructions





##### Automatic testing

* Run `npm run test:js multi-currency-toggle``

##### Manual testing 

* Open `WooCommerce > Payments` page
* Scroll down to see the Advanced link, click it
* On the revealed form, uncheck the `Enable customer multi currency` toggle
* Save the changes
* Refresh the page and check if the *Multi-Currency* tab is still visible. It should be removed.
* On the same form, enable the checkbox back and save.
* Refresh the page and expect the *Multi-Currency* tab is back in its place again.

-------------------


**Post merge**
